lib/modules/apostrophe-schemas/index.js

```diff
@@ -1867,6 +1867,12 @@ module.exports = {
       name: 'date',
       converters: {
         string: function(req, data, name, object, field, callback) {
+          if (data[name] === '') {
```

Comment:
This is good, but actually it could also be `null` as it could be a JSON submission. Indeed a simple check for a falsy `date[name]` would work much better than usual here.